Api/src/controllers/userController.js

```diff
@@ -0,0 +1,88 @@
+// import {
+//   v4 as uuidv4
+// } from 'uuid';
+
+import data from "../data.json" assert { type: "json" };;
+
+// let users = data.Users;
+let users = [];
+
+export const getUsers = (req, res) => {
+  res.send({
+    status: 200,
+    users: data.Users
+  });
+};
+
+export const createUser = (req, res) => {
+  // id: uuidv4(),
+
+  const user = {
+    id: req.body.id,
+    firstName: req.body.firstName,
```

Comment:
We're calling `req.body` too many times which is repetitive. We should usually prefer destructuring.

```
const { id, firstName, lastName, ... } = req.body;
```


NB: Update other places in your changes that do this repetition.

Reply:
Okay, thank you.

Reply:
you instead did the opposite and reverted to the way it was initially. Meanwhile, you are doing this correctly in other places.

Reply:
Followed the same spec it should be good now.